README.md

```diff
@@ -0,0 +1,80 @@
+# MCR Codes Music Library
+
+This is my music library project for [Manchester Codes](https://www.manchestercodes.com/) as part of the Backend module.
+
+## Description
+
+The main purpose of this project is to learn how to design and implement an API which can perform CRUD operations on a database.
+
+## Features
+
+Use RESTful routes to perform CRUD operations to store and manage your favourite music artists in a database.
+
+## Requirements
+
+- [Node JS](https://nodejs.org/en/) - to start the server.
+
+### Application Dependencies
+
+- [Express](https://expressjs.com/) - to set up the server and manage routes and responses.
+- [MySQL2](https://github.com/sidorares/node-mysql2#readme) - to work with the MySQL database engine.
+
+### Development Dependencies
+
+- [Nodemon](https://nodemon.io/) - to run the 'npm start' script
+- [Dotenv](https://github.com/motdotla/dotenv#readme) - to store environment variables
+- [Mocha](https://mochajs.org/) - testing framework
+- [Chai](https://www.chaijs.com/) - assertion library
+- [SuperTest](https://github.com/visionmedia/supertest#readme) - http assertions with async/await
+
+## Getting Started
+
+- `docker pull bbatm9/music_library_mysql` more info: [DockerHub](https://hub.docker.com/r/bbatm9/music_library_mysql)
```

Comment:
This did not work. I can see the repo on the docker hub, sonot sure why this didnt work off hand. I got it running using MAMP to host the database instead, but would be good to understand why this didnt work as expected

Reply:
Hmm, I ran `docker image ls` and the image shows as simply `mysql` rather than `music_library_mysql` as I would have expected, did you also pass `music_library_mysql` as the image name when you tried to run docker run?